### PR TITLE
[DEV-2784] add payment routes

### PR DESCRIPTION
**Pull request by lapatric**
- [DEV-2784]  add payment routes to services

[DEV-2784]: https://dfxswiss.atlassian.net/browse/DEV-2784?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/screens/payment-routes.screen.tsx
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:1]: Remove logs

File: src/screens/payment-routes.screen.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+              <StyledDataTableExpandableRow
+                label={`${route.asset.blockchain} / ${route.asset.name}`}
+                expansionItems={[
+                  { label: translate('screens/payment', 'Asset'), text: route.asset.name },
+                  { label: translate('screens/payment', 'Blockchain'), text: route.asset.blockchain },
+                  { label: translate('screens/payment', 'Purpose of payment'), text: route.bankUsage },
+                  {
+                    label: translate('screens/payment', 'Minimum deposit'),
+                    text: `${route.minDeposits.map((d) => `${d.amount} ${d.asset}`).join(', ')}`,
+                  },
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:107]: Can be removed

File: src/screens/payment-routes.screen.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            {routes.sell.map<JSX.Element>((route) => (
+              <StyledDataTableExpandableRow
+                label={`${route.currency.name} / ${route.iban}`}
+                expansionItems={[
+                  { label: translate('screens/payment', 'Currency'), text: route.currency.name },
+                  { label: translate('screens/payment', 'IBAN'), text: route.iban },
+                  {
+                    label: translate('screens/payment', 'Deposit address'),
+                    text: `${route.minDeposits.map((d) => `${d.amount} ${d.asset}`).join(', ')}`,
+                  },
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:135]: Deposit address or min deposit? 😆 
Min. deposit can be removed

File: src/screens/payment-routes.screen.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+                  {
+                    label: translate('screens/payment', 'Volume'),
+                    text: `${route.annualVolume} CHF`,
+                  },
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:139]: Volume or annual volume?

File: src/screens/payment-routes.screen.tsx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+            {routes.swap.map<JSX.Element>((route) => (
+              <StyledDataTableExpandableRow
+                label={route.asset.name}
+                expansionItems={[
+                  { label: translate('screens/payment', 'Asset'), text: route.asset.name },
+                  { label: translate('screens/payment', 'Blockchain'), text: route.asset.blockchain },
+                  {
+                    label: translate('screens/payment', 'Minimum deposit'),
+                    text: `${route.minDeposits.map((d) => `${d.amount} ${d.asset}`).join(', ')}`,
+                  },
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:160]: Can be removed

File: src/screens/payment-routes.screen.tsx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+                    text: `${route.volume} CHF`,
+                  },
+                  {
+                    label: translate('screens/payment', 'Annual volume'),
+                    text: `${route.annualVolume} CHF`,
+                  },
+                ]}
+              >
+                {route.asset.name}
+              </StyledDataTableExpandableRow>
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:172]: Show deposit address and blockchains (see sell)

File: src/screens/payment-routes.screen.tsx [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+  fee: number;
+  minDeposits: MinAmount[];
+  minFee: MinAmount;
+}
+
+export interface SellRoute {
+  id: number;
+  active: boolean;
+  deposit: any;
+  iban: string;
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:42]: `DepositDto`?

File: src/screens/payment-routes.screen.tsx
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:1]: Only show active routes

**Review by lapatric — COMMENTED** [review]
- [ ] Refactor: Move type definitions and API functions to packages

File: src/screens/payment-routes.screen.tsx [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+                    },
+                  ]}
+                >
+                  <p className="text-right overflow-ellipsis">
+                    {route.deposit.blockchains.map((blockchain: Blockchain) => toString(blockchain)).join(', ')}
+                  </p>
+                </StyledDataTableExpandableRow>
+              ))}
+            </StyledDataTable>
+          )}
```
Comment (davidleomay) [inline src/screens/payment-routes.screen.tsx:149]: Deposit address and volume are missing

**Review by davidleomay — APPROVED** [review]

**Review by davidleomay — APPROVED** [review]

**Pull request merged**